src/tx/sendTx.js

```diff
@@ -51,6 +51,12 @@ async function sendRawTx({ chain, params, method }) {
         id: Math.floor(Math.random() * 100) + 1
       })
     })
+
+    if (!response.ok) {
```

Comment:
Could you share more details why this is needed now? Why did it work before?

Reply:
The RPC endpoint always returns a 200 status code, even if the transaction failed (the error is inside the response body). But this could fail for connection reasons: for example, if the RPC is behind a proxy and the proxy returns a 404 for some reason.

I think this worked before because we used local or reliable endpoints, but when I checked with failing URLs the lack of this check was causing issues.

Reply:
thanks for explanation. I am OK with this.